components/econet/econet.h

```diff
@@ -147,10 +147,10 @@ class Econet : public Component, public uart::UARTDevice {
   std::vector<std::set<std::string>> request_datapoint_ids_ = std::vector<std::set<std::string>>(MAX_REQUEST_MODS);
   std::vector<uint32_t> request_mod_last_requested_ = std::vector<uint32_t>{MAX_REQUEST_MODS, 0};
   uint8_t request_mods_{1};
-  std::set<std::string> raw_datapoint_ids_;
-  std::set<std::string> request_once_datapoint_ids_;
-  std::map<std::string, EconetDatapoint> datapoints_;
-  std::map<std::string, EconetDatapoint> pending_writes_;
+  std::set<std::pair<std::string, uint32_t>> raw_datapoint_ids_;
```

Comment:
Create a struct instead of a pair and use it in all these 4 places. It's more readable and it's more future proof.